src/installer.ts

```diff
@@ -76,15 +77,105 @@ export async function getJuliaVersions(versionInfo): Promise<string[]> {
     return versions
 }
 
-export function getJuliaVersion(availableReleases: string[], versionInput: string, includePrerelease: boolean = false): string {
+export function getProjectFile(projectInput: string = ""): string {
```

Comment:
Might be good to rename this something like `getProjectFilename`? To emphasize that it's only returning the filename, and not the file contents?

Reply:
Function was renamed to `getProjectFilePath`